hasher-matcher-actioner/hmalib/common/models/bank.py

```diff
@@ -580,6 +589,24 @@ def add_detached_bank_member_signal(
             signal_value=signal_value,
         )
 
+    def remove_bank_member_signals_to_process(self, bank_member_id: str):
+        """
+        For a bank_member, remove all signals from the processing index.
```

Comment:
nit: expand doc-string with more info and/or where to look, the to process index is complex enough this doc string could be misleading.